src/Queue/EventQueueConsumer.php

```diff
@@ -1,26 +1,21 @@
 <?php declare(strict_types=1);
 
-namespace Comquer\Event\Queue;
+namespace Comquer\Queue;
 
 use Comquer\BusException;
 use Comquer\Event\Bus\EventBus;
 use Comquer\Event\Store\EventStore;
-use Comquer\Queue\Queue;
 use Comquer\Queue\QueueConsumer;
 
-class EventQueueConsumer implements QueueConsumer
+class EventQueueConsumer
 {
-    private $eventStore;
-
-    private $eventBus;
-
-    public function __construct(EventStore $eventStore, EventBus $eventBus)
+    public function __construct()
     {
         $this->eventStore = $eventStore;
         $this->eventBus = $eventBus;
     }
 
-    public function consume(Queue $queue): void
+    public function consume(EventListenerQueue $queue): void
     {
         while ($queue->isEmpty() === false) {
             $event = $this->eventStore->getById($queue->pullNext());
```

Comment:
This class uses the deleted EventBus class. Should it use the EventPublisher instead?